migration/sqls/000007_add_timeout_to_job.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `job`
+ADD COLUMN `timeout` INTEGER NOT NULL DEFAULT 0;
```

Comment:
Are we trying to save `time.Duration` (which is in nanoseconds) into an INTEGER field in DB?
I'm getting integer overflow for values higher than 2147483647 (2^31-1). So the limit seems to be 2.147... seconds?
Or am I doing something wrong here 😢

Reply:
That is a good question; we should simplify it and just save timeout in seconds, so using Integer all the way through and though is what we should do. What this also brings to light is, we should probably add a integration test with _240s_ timeout to ensure it sets properly.

Reply:
Bhaia, I have added an integration test with this behavior. A job is put in-flight with `incrementalTimeout` value of 200s. And, after 300s (a loose time delay) that job is put in `JobQueued` status back again.

Reply:
The integration test should not wait for 300s; that is not needed; we can set the default timeout to 5s; and custom to 25s; and test at 15s mark if we need to.